### DISCUSSION TITLE
What's the proper way to integrate with Home Assistant when Frigate is hosted on a separate server?

### DISCUSSION THREAD

**Discussion by szaske** [blakeblackshear/frigate#2744] [Ask A Question]
I run Frigate on a separate box...not on my Home Assistant server. I also want to integrate with Frigate from Node Red. If I install the Frigate Integration it seems to work, but I get errors in my logs that say 
"Watchdog found a problem with ccab4aaf_frigate-proxy application!"...but I'm still able to integrate with Frigate via Node Red using devices like binary_sensor.door_person_motion.

I've also installed the "Frigate Proxy". am I only supposed to use EITHER the Integration or the Proxy and not both at the same time?

If I run Frigate on its own server am I then required to use MQTT to use it within Node Red or can I still install the Integration?

**Comment by blakeblackshear** [comment]
You can install both. The proxy just makes frigate's interface available in the sidebar. The integration provides everything else. It sounds like you pointed the integration at the proxy addon's address. You should be using the direct address for the box running frigate in the integration setup.

You always need an mqtt server. As long as everything is connected to the same mqtt server, everything should work together.
